sktime/forecasting/naive.py

```diff
@@ -23,13 +24,21 @@ class NaiveForecaster(OptionalForecastingHorizonMixin,
 
     Parameters
     ----------
-    strategy : str{"last", "mean", "seasonal_last"}, optional (default="last")
+    strategy : str{"last", "mean"}, optional (default="last")
         Strategy used to make forecasts:
 
-        * "last": forecast the last value in the training series
-        * "mean": forecast the mean of (a given window) of the training series
-        * "seasonal_last": forecasts the last value of the same season in
-        the training series
+        * "last" and sp is None: forecast the last value in the
```

Comment:
perhaps it's easier to simply state what the strategies "last" and "mean" do and then say that when sp is also given (i.e. when sp is not 1), that we then compute the mean and last value based on the values of the same season